### ISSUE TITLE
Which nightly compiler version is needed?

### ISSUE THREAD

**Issue by ghuls** [linux-china/zawk#3]
Which nightly compiler version is needed?

**Comment by ghuls** [comment]
```
$ cargo +nightly install --path . --no-default-features --features use_jemalloc,allow_avx2,unstable
...
      Adding windows_x86_64_msvc v0.48.5 (available: v0.52.6)
      Adding winreg v0.50.0 (available: v0.52.0)
      Adding wyz v0.5.1 (available: v0.6.1)
      Adding xmltree v0.10.3 (available: v0.11.0)
   Compiling zawk v0.5.17 (/software/zawk)
error[E0658]: use of unstable library feature 'core_intrinsics': intrinsics are unlikely to ever be stabilized, instead they should be used through stabilized interfaces in the rest of the standard library  
    --> src/runtime/splitter/batch.rs:1140:17                                                                                                                                                                  
     |                                                                                                                                                                                                         
1140 |                 std::intrinsics::prefetch_read_data($buf.offset(128), 3);                                                                                                                               
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                     
...                                                                                                                                                                                                            
1151 |                     let (rel, nl) = iterate!(buf_ptr.add(V::INPUT_SIZE * b + ix));                                                                                                                      
     |                                     --------------------------------------------- in this macro invocation                                                                                              
     |                                                                                                                                                                                                         
     = help: add `#![feature(core_intrinsics)]` to the crate attributes to enable                                                                                                                              
     = note: this compiler was built on 2024-09-19; consider upgrading it if it is out of date                                                                                                                 
     = note: this error originates in the macro `iterate` (in Nightly builds, run with -Z macro-backtrace for more info)                                                                                       
                                                                                                                                                                                                               
error[E0658]: use of unstable library feature 'core_intrinsics': intrinsics are unlikely to ever be stabilized, instead they should be used through stabilized interfaces in the rest of the standard library  
    --> src/runtime/splitter/batch.rs:1140:17                                                                                                                                                                  
     |                                                                                                                                                                                                         
1140 |                 std::intrinsics::prefetch_read_data($buf.offset(128), 3);                                                                                                                               
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                     
...                                                                                                                                                                                                            
1175 |             let (fields, nl) = iterate!(buf_ptr.add(ix));                                                                                                                                               
     |                                ------------------------- in this macro invocation                                                                                                                       
     |                                                                                                                                                                                                         
     = help: add `#![feature(core_intrinsics)]` to the crate attributes to enable                                                                                                                              
     = note: this compiler was built on 2024-09-19; consider upgrading it if it is out of date                                                                                                                 
     = note: this error originates in the macro `iterate` (in Nightly builds, run with -Z macro-backtrace for more info)                                                                                       
                                                                                                                                                                                                               
error[E0658]: use of unstable library feature 'core_intrinsics': intrinsics are unlikely to ever be stabilized, instead they should be used through stabilized interfaces in the rest of the standard library  
    --> src/runtime/splitter/batch.rs:1140:17   
...
```

**Comment by linux-china** [comment]
Just use `stable` channel, nightly not required.
